packages/contracts-bedrock/test/vendor/Initializable.t.sol

```diff
@@ -396,6 +396,7 @@ contract Initializer_Test is CommonTest {
         excludes[j++] = "src/dispute/SuperFaultDisputeGame.sol";
         excludes[j++] = "src/dispute/PermissionedDisputeGame.sol";
         excludes[j++] = "src/dispute/SuperPermissionedDisputeGame.sol";
+        excludes[j++] = "src/dispute/zk/OPSuccinctFaultDisputeGame.sol";
```

Comment:
Could we add an issue to fix the contracts and remove this exception again.  Totally fine to have it for now since we're importing directly but given these are new contracts we're importing we should try to make them follow best practice if we can and I don't think we identified any initializable work previously.